### PR TITLE
Documentation for home-assistant/home-assistant-polymer#1745

### PR DESCRIPTION
**Pull request by thomasloven**
**Description:**
Documentation for the click-and-hold feature for lovelace picture-elements

**Pull request in [home-assistant-polymer](https://github.com/home-assistant/home-assistant-polymer) (if applicable):** home-assistant/home-assistant-polymer#1745

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_lovelace/picture-elements.markdown [hunk 1]
```diff
@@ -74,6 +74,11 @@ tap_action:
   description: more-info, toggle, navigate, call-service
   type: string
   default: more-info
+hold_action:
+  required: false
+  description: Action to perform when clicked-and-held. more-info, toggle, navigate, call-service
```
Comment (arsaboo) [inline source/_lovelace/picture-elements.markdown]: Action to perform when clicked-and-held (e.g., `more-info`, `toggle`, `navigate`, `call-service`)

Comment (frenck) [reply source/_lovelace/picture-elements.markdown]: Done. thx :)

**Review by frenck — APPROVED** [review]
✅ Approved. Made some small changes and can be merged as soon as the build passes.

**Comment by thomasloven** [comment]
Thanks for fixing, @frenck!

**Pull request merged**